checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -123,29 +123,28 @@ protected void checkThisOrSuperConstructorCall(
     @Override
     protected boolean commonAssignmentCheck(
             Tree varTree,
-            ExpressionTree valueExp,
+            ExpressionTree valueExpTree,
             @CompilerMessageKey String errorKey,
             Object... extraArgs) {
         // field write of the form x.f = y
         if (TreeUtils.isFieldAccess(varTree)) {
             // cast is safe: a field access can only be an IdentifierTree or MemberSelectTree
             ExpressionTree lhs = (ExpressionTree) varTree;
-            ExpressionTree y = valueExp;
             VariableElement el = TreeUtils.variableElementFromUse(lhs);
-            AnnotatedTypeMirror xType = atypeFactory.getReceiverType(lhs);
-            AnnotatedTypeMirror yType = atypeFactory.getAnnotatedType(y);
+            AnnotatedTypeMirror lhsType = atypeFactory.getReceiverType(lhs);
```

Comment:
This uses `getReceiver`, so it's not the type of the `lhs`, is it? It's the receiver of the LHS?

Reply:
okay, renamed to `lhsReceiverType`.